src/book/03-guides/03-hardware/04-DARwIn-OP2-guide.mdx

```diff
@@ -0,0 +1,184 @@
+slug: /guides/hardware/DARwIn-OP2-guide
+authors:
+  - Corah Oliver (@yippyy)
+---
+
+## DARwIn OP2 Guide
+
+### DARwIN OP2 Specifications
+
+- Mainboard: PICO-CV01
+- Cpu: Intel Atom N2600 @1.6GHz single core
+- Ram: Up to 4GB DDR3/240-ping SO-DIMM module (user replaceable)
+- Storage: Half-size mSATA module (32GB) (user replaceable)
+- Sub controller: CM-740
+- Linux mint 17.3 'Rosa' installed on PICO-CV01 board
+
+### Creating the recovery USB
+
+1. Download Tuxboot from the [SourceForge website](https://sourceforge.net/projects/tuxboot/files/0.6/). For this demonstration we will be downloading the Linux version.
+   ![Downloading Tuxboot 1](./images/DARwIn_OP2_guide/Tuxboot1.jpg)
+   ![Downloading Tuxboot 2](./images/DARwIn_OP2_guide/Tuxboot2.jpg)
+
+2. When the first step is complete, run Tuxboot 0.6
+   ![Running Tuxboot](https://emanual.robotis.com/assets/images/platform/op/op_039.jpg 'Photo from Robotis')
+
+3. Using this [Link](https://sourceforge.net/projects/clonezilla/) will take you to the following screen
+   ![Clonezilla 1](./images/DARwIn_OP2_guide/Tuxboot4.jpg)
+
+Click onto the 'Files' tab, then select 'clonezilla_live_stable'.
+![Clonezilla 2](./images/DARwIn_OP2_guide/Tuxboot8.jpg)
+
+This will then direct you to a tab filled with different clonezilla versions. What we are looking for is the 'OldFiles' section.
+![Clonezilla 3](./images/DARwIn_OP2_guide/Tuxboot5.jpg)
+
+Clicking into this, we now need to look for the section called '2.0.1-5' (in the following picture, it is the middle option)
+![Clonezilla 4](./images/DARwIn_OP2_guide/Tuxboot6.jpg)
+
+In this folder, we need to download 'clonezilla-live-2.0.1-5-i486.iso' (in the following picture, it is the middle option)
+![Clonezilla 5](./images/DARwIn_OP2_guide/Tuxboot7.jpg)
+
+4. When you have successfully downloaded Clonezilla, reopen Tuxboot. The select the 'pre=downloaded' option on screen.
+   Select the 'clonezilla-live-2.0.1-5-i486.iso' from your files. Make sure you have ISO selected in the drop-down menu, for the 'Type' selection, pick USB and tick the MD5 check box.
+   ![Clonezilla 6](https://emanual.robotis.com/assets/images/platform/op/op_041.jpg 'Photo from Robotis')
+
+After you have done this step, press 'OK' and let the program run.
+![Clonezilla 7](https://emanual.robotis.com/assets/images/platform/op/op_043.jpg 'Photo from Robotis')
+
+Once this is complete press 'Exit'.
+![Clonezilla 8](https://emanual.robotis.com/assets/images/platform/op/op_044.jpg 'Photo from Robotis')
+
+This completes the Clonezilla portion of this recovery USB.
+
+**Now that you have successfully created a Clonezilla USB, we now need to put the recovery software onto it. This is done by following the instructions below**
+
+5. Open your USB in your files but leave this for now and following the following [Link](https://sourceforge.net/projects/darwinop/files/Software/Main%20Controller/Recovery%20USB/)
```

Comment:
```suggestion
5. Open your USB in your files but leave this for now and [go to SourceForge](https://sourceforge.net/projects/darwinop/files/Software/Main%20Controller/Recovery%20USB/)
```
or something similar, tell the user what the link is before they click it. same thing with your other links